libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2549,61 +2571,66 @@ bool TypeChecker::visit(MemberAccess const& _memberAccess)
 					" outside of storage."
 				);
 		}
-		string errorMsg = "Member \"" + memberName + "\" not found or not visible "
-				"after argument-dependent lookup in " + exprType->toString() + ".";
 
-		if (auto const& funType = dynamic_cast<FunctionType const*>(exprType))
-		{
-			auto const& t = funType->returnParameterTypes();
+		auto [errorId, description] = [&]() -> tuple<ErrorId, string> {
+			string errorMsg = "Member \"" + memberName + "\" not found or not visible "
+				"after argument-dependent lookup in " + exprType->toString() + ".";
 
-			if (memberName == "value")
+			if (auto const& funType = dynamic_cast<FunctionType const*>(exprType))
```

Comment:
I know it is old code, but should be fixed to:
```suggestion
			if (auto const* funType = dynamic_cast<FunctionType const*>(exprType))
```